webpack.config.js

```diff
@@ -0,0 +1,35 @@
+const { resolve } = require('path') 
+const MiniCssExtractPlugin = require('mini-css-extract-plugin') 
+const HtmlWebpackPlugin = require('html-webpack-plugin') 
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer') 
+
+module.exports = {
+    mode: 'development',
+    entry: "./js/main.js",
+    output: {
+        path: resolve(__dirname, 'build'), 
+        filename: "main.[contenthash].js"
+    },
+
+    module: {
+        rules: [
+            {
+                test: /\.(png|jpe?g|gif)$/i,
```

Comment:
чтобы в сборку включался еще и звук сюда надо добавить расширение mp3
```suggestion
                test: /\.(png|jpe?g|gif|mp3)$/i,
```